index.bs

```diff
@@ -306,6 +306,10 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "S
     [=credential key pair=] is known as the <dfn>credential private key</dfn>. Note that in the case of [=self
     attestation=], the [=credential key pair=] is also used as the [=attestation key pair=], see [=self attestation=]
     for details.
+ 
+: <dfn>Rate Limiting</dfn>
+:: Rate limiting is the process by which an authenticator limits the number of unsuccessful authentication attemps within a set
```

Comment:
"Authentication attempts" are a term-of-art.  See https://pages.nist.gov/800-63-3/sp800-63b.html#throttle.